src/pages/home/report/ReportActionContextMenu.js

```diff
@@ -130,7 +130,7 @@ class ReportActionContextMenu extends React.Component {
             {
                 text: this.props.translate('reportActionContextMenu.deleteComment'),
                 icon: Trashcan,
-                shouldShow: this.isUserActorOfAction(),
+                shouldShow: () => this.canEdit(),
```

Comment:
NAB

```suggestion
                shouldShow: this.canEdit,
```

Reply:
only able to change to `shouldShow: () => this.canEdit. Removing the `() =>` breaks the app

Reply:
That doesn't sound right... It should be totally possible to just pass a reference to `this.canEdit` rather than wrapping it in an arrow function.